share/spice/open_nic/open_nic.js

```diff
@@ -0,0 +1,28 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_open_nic = function(api_result){
+
+        if (!api_result) {
+            return Spice.failed('open_nic');
+        }
+
+        Spice.add({
+            id: "open_nic",
+            name: "DNS",
+            data: {
+                list: api_result
+            },
+            meta: {
+                sourceName: "OpenNIC",
+                sourceUrl: 'https://api.opennicproject.org/geoip/'
```

Comment:
Sorry about the back and forth, but I think the previous URL was more helpful. Typically we try to link to a search for the original query at the source website, when applicable so they can see more relevant info. In this case a page explaining OpenNIC or that gives a little more context about the IP's we're showing would be ideal.